tools/tsp-client/test/run_commands.ts

```diff
@@ -78,37 +78,40 @@ async function main() {
   // Run the convert command on a mgmt spec
   await runCommand(baseDir, ["convert", "-o", baseSpecDir, "--swagger-readme", mgmtSpecLink]);
 
+  // Run the compare command to ensure it is working
+  // TODO: Provide actual lhs and rhs values
```

Comment:
Maybe you could just copy/paste the contoso widget manager generated tsp and add a couple of the handwritten modifications that you know are typically made in a handwritten swagger?

Reply:
I copied in the files from one of my manual tests.